testcharms/charm-repo/bionic/dummy-no-series/metadata.yaml

```diff
@@ -0,0 +1,6 @@
+name: dummy
+summary: "That's a dummy charm."
+description: |
+    This is a longer description which
+    potentially contains multiple lines.
+minjujuversion: "1.0.0"
```

Comment:
This is not a valid charm. It must have a manifest.yaml, or worst case the series specified in the metadata.yaml

Reply:
I agree, this charm was created because some tests assert that such charms (without series in metadata or a manifest) fail to deploy

Reply:
Where is that documented?

Reply:
Not sure. Pretty sure you've told me before that charms must define an OS in either metadata or manifest to be valid though

Reply:
Correct. Please add a comment in the charm to say it's for testing a bad charm and should not be used as a basis for a good charm. Maybe a name of failing-charm-no-os?

Reply:
Done